### ISSUE TITLE
Using datatables with sheetrock yields Uncaught TypeError

### ISSUE THREAD

**Issue by jakob1111** [chriszarate/sheetrock#143]
When I try to show my Google sheet as a datatable the table is shown but not as a datatable. My code looks as follows:

```
<script>
var mySpreadsheet = 'https://docs.google.com/spreadsheets/d/1OLMSsCiPgN72i929Ue5wTUJRdZM4qfNOCbqQFuFJHJY/edit#gid=0';

$('#educators').sheetrock({
  url: mySpreadsheet,
  callback: makeSortable

});

function makeSortable(error, options, response) {
  // Target is not a jQuery object, so we need to call jQuery
  // explicitly.
  $(options.user.target).DataTable();
}
</script>
```

The error I get reads:

`jquery.dataTables.js:1197 Uncaught TypeError: Cannot read property 'mData' of undefined`

The problem seems to be that sheetrock does not produce a proper table header for the google sheet in question.  However, I have no idea what I need to change in the Google sheet to make it work properly. (I tried freezing the first row and making it bold but the problem persists.)

**Comment by leolamanna** [comment]
Same problem here. Sheetrock working PERFECT. DataTables working PERFECT... but when I mix them, DataTables WON´T WORK. Any help?? Pleaseeee?  =(

**Comment by chriszarate** [comment]
> sheetrock does not produce a proper table heade

What does that mean in HTML terms? Do `<th>` need to be wrapped in `<thead>`?

**Comment by chriszarate** [comment]
Fixed in `v1.2.0`. Example using DataTables:

https://chriszarate.github.io/sheetrock/#datatables
